Use a different command for the docker compose v2 support

nbrinckm opened the pull request:
If the `docker compose version` succeeds, then use `docker compose`, otherwise use `docker-compose`.

nbrinckm commented:
This will make a better test, but I'm not 100% sure if the run function in the build script supports muitple docker-compose files then.

MichaelLangbein reviewed (approved):
Nothing to add here - works perfectly fine. Approved :)

MichaelLangbein commented:
> This will make a better test, but I'm not 100% sure if the run function in the build script supports muitple docker-compose files then.

Works with v2 (docker compose) - does it work alright with v1 (docker-compose)?

nbrinckm commented:
> > This will make a better test, but I'm not 100% sure if the run function in the build script supports muitple docker-compose files then.
> 
> Works with v2 (docker compose) - does it work alright with v1 (docker-compose)?

Doesn't seem so out of the box...

Changes I did so far:
- I added a block:
```
env_file
- env
```
to the services.

- I added a version to the docker-compose.dlr.yaml file
- I removed the dynamic port mapping & made it static.
